storage_service/locations/models/dspace.py

```diff
@@ -365,39 +365,39 @@ def _set_permissions(self, package):
         params = {'expand': 'bitstreams'}
         try:
             response = requests.get(url, headers=headers, params=params)
+            item = response.json()
         except Exception:
             LOGGER.warning('Error fetching bitstream information for handle %s', handle, exc_info=True)
         LOGGER.debug('REST API handle mapping %s %s', response.status_code, response)
         LOGGER.debug('Body %s', response.json())
```

Comment:
Small detail, we could use `item` in here now, to avoid transforming the response twice.